docs/scripts/get.sh

```diff
@@ -5,19 +5,23 @@ if [ -x "$(command -v docker)" ]; then
     mkdir websight-cms-ce
```

Comment:
It might be a good idea to add shebang, e.g. `#!/bin/sh` at the beginning of the script.